Promact.Oauth.Server/src/Promact.Oauth.Server/Startup.cs

```diff
@@ -27,9 +27,10 @@ namespace Promact.Oauth.Server
     public class Startup
     {
         private ILoggerFactory _loggerFactory { get; }
-
+        private IHostingEnvironment CurrentEnvironment { get; set; }
```

Comment:
public Startup(IHostingEnvironment env, ILoggerFactory loggerFactory)

I think It is